threema/gateway/e2e.py

```diff
@@ -659,10 +659,15 @@ class DeliveryReceipt(Message):
     """
     @enum.unique
     class ReceiptType(enum.IntEnum):
-        """Describes message receipt types."""
+        """
+        Describes message receipt types.
+
+        .. warning:: `user_ack` is deprecated and will be removed with
+           the next major release. Use `user_acknowledge` instead.
```

Comment:
Had the same idea but that would have to be a class property and class properties require some fiddling. But feel free to add it.